src/transformers/pipelines/token_classification.py

```diff
@@ -85,6 +85,7 @@ def __init__(
         ignore_labels=["O"],
         task: str = "",
         grouped_entities: bool = False,
+        subword_label_re_alignment: Union[bool, str] = False,
         ignore_subwords: bool = False,
```

Comment:
Should the `ignore_subwords` flag be removed then?

Reply:
I think so, we should remove `ignore_subwords` flag. @LysandreJik, @Narsil  i leave some of the old codes just to pass the tests(😅 i'm new to tests). Can you help me about the tests? ( for example should I remove [this](https://github.com/huggingface/transformers/blob/ce9724e1bdf3cd84a3b50f771d35b557629e95fc/tests/test_pipelines_ner.py#L316) test or somehow change it ? )

Reply:
It would be nice to keep it for backwards compatibility purposes. Can the capabilities enabled by that flag be achieved with the new flag introduced in this PR?